osmaxx-py/osmaxx/excerptexport/templates/excerptexport/templates/access_denied.html

```diff
@@ -18,7 +18,7 @@
                     {% csrf_token %}
                     <button class="btn btn-primary"><span class="icon default">&#9911;</span> {% trans 'Request access' %}</button>
                 </form>
-                {% blocktrans with next=next_page %}and try <a href="{{ next_page }}">again</a>.{% endblocktrans %}
+                {% blocktrans with next=next_page %}and try again.{% endblocktrans %}
```

Comment:
If the placeholder `{{ next_page }}` is gone, the value for it (`with ...`) can be removed, too, I guess?